src/seedu/addressbook/commands/Command.java

```diff
@@ -54,6 +54,13 @@ public void setData(AddressBook addressBook, List<? extends ReadOnlyPerson> rele
         this.relevantPersons = relevantPersons;
     }
 
+    /**
+     * Checks for command types that mutate the data. The command returns false by default.
```

Comment:
Good effort in adding header comments for `isMutating`.

Though it is not good for `isMutating` in `Command` to have a default implementation.

Having a default implementation for the `isMutating` method may not be intuitive for a class like `Command` as it is abstract in context. Why return false and not true?

Perhaps a better implementation would be make this method abstract and force all classes extended from `Command` to implement their own method definition.